### PR TITLE
Upgrades epoll crate. 

### PR DESCRIPTION
**Pull request by nathansizemore**
Upgraded from 2.1.0 to 4.0.1. Implements updates from nathansizemore/epoll#8

#757 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by nathansizemore** [comment]
Thought it was cool you all were using my crate, so I went ahead and upgraded it to latest version for you to help with the aarch64 stuff. Lemme know if I need to do anything else!

File: vmm/Cargo.toml [hunk 1]
```diff
@@ -33,4 +33,3 @@ tempfile = ">=3.0.2"
 
 [features]
 vsock = ["devices/vsock"]
-
```
Comment (andreeaflorescu) [inline vmm/Cargo.toml]: nit: can you leave this line as it was?

**Comment by andreeaflorescu** [comment]
You will also need to update the tests. Check the full log here: https://firecracker-pr.s3.amazonaws.com/pr-logs/00787/nathansizemore-firecracker-6dc96a2.log

**Comment by dianpopa** [comment]
Thanks, @nathansizemore ! You read my mind with this PR! Will add my review asap.

File: vmm/Cargo.toml (hunk 1 above)
Comment (nathansizemore) [reply vmm/Cargo.toml]: Trim whitespace on save strikes again. Fixed in latest commit, along with signature.

**Pull request merged**